packages/web/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web",
   "version": "0.0.1",
   "scripts": {
-    "start": "cross-env PORT=8080 react-scripts start",
+    "start": "cross-env HTTPS=true PORT=8080 react-scripts start",
```

Comment:
redundant

Reply:
let's have a separate command to run dev server over HTTP and use a separate port (e.g. localhost:4430, same pair of ports we use on the server). We also need additional environments config for it (e.g. localHttps). Otherwise, with the current config we will have https FE trying to connect to http BE, which should fail with an error.